manifests/pod.yaml

```diff
@@ -42,9 +42,7 @@ spec:
        privileged: false
      resources:
        requests:
-         # Note the supermin VM just uses 2G. The really hungry part is xz,
-         # which without lots of memory takes lots of time.
-         memory: 6.5Gi
```

Comment:
previously this value was slightly larger than the actual memlimit. i.e. 6.5GiB vs 6144MiB. Now they would be the same. Is that OK?

Reply:
OK, I just changed this to do the `export` from the shell so we can drop it down a bit. Also, doing some local testing on `xz` shows that for the artifact sizes we're compressing, it shouldn't need more than 5.5G. So I just lowered it to 6.5G again.